go/common/config.go

```diff
@@ -279,6 +326,22 @@ func LoadGlobalConfig(path string) error {
 		return err
 	}
 
+	// Ensure mem_pool_mb meets the minimum based on current limits.
```

Comment:
This is pretty confusing to me, and doesn't seem like it belongs here.  How do you know the machine has enough RAM to bump it up?  Even if they do, if they set a low mem pool size, they probably are reserving memory for some other system being deployed alongside OpenLambda.  We should inform the user (as checkConf does), but don't second guess them.